validate-differential-expression/author_processed_DE.Rmd

```diff
@@ -0,0 +1,158 @@
+---
+title: "Differential Expression Analysis for GSE37382"
+author: "ALSF CCDL - Candace Savonen"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+*Purpose*: This notebook tests differential expression in probe data from 
+author-submitted version of GSE37382, a medulloblastoma microarray dataset. 
+
+The results from this differential expression analysis is used in the following
+notebook to compare how the processing of this microarray data affects 
+differential expression outcomes.
+The data here are medulloblastoma microarray data produced by [Northcott et al, 2012](https://www.ncbi.nlm.nih.gov/pubmed/22832581) and are obtained from Gene 
+Expression Omnibus (GEO).
+
+## 1) Install libraries
+This script uses the bioconductor R package limma to identify differentially 
+expressed genes.  
+The full guide on [limma](https://bioconductor.org/packages/release/bioc/vignettes/limma/inst/doc/usersguide.pdf) 
+shows examples of limma functions.  
+We will also use the library `GEOquery` to obtain the author-processed data. 
+
+```{r Install limma}
+if (!("limma" %in% installed.packages())) {
+  # Install limma
+  BiocManager::install("limma", update = FALSE)
+}
+if (!("GEOquery" %in% installed.packages())) {
+  # Install GEOquery
+  BiocManager::install("GEOquery", update = FALSE)
+}
+```
+
+Attach the `limma` library:
+
+```{r}
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Attach library
+library(limma)
+```
+
+Set up output directories.
+
+```{r}
+# Make a results directory if it isn't created yet
+if (!dir.exists("results")) {
+  dir.create("results")
+}
+# Make a plots directory if it isn't created yet
+if (!dir.exists("plots")) {
+  dir.create("plots")
+}
+```
+
+## 2) Import and set up author processed data from GEO
+For more information on how this data was processed, see its 
+[GEO entry](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37382)
+
+```{r}
+# Store file name for gene matrix
+matrix.file.name <- file.path("data", "GSE37382_series_matrix.txt")
+
+# Download author processed data file directly to `data` folder
+geo.data <-  GEOquery::getGEO("GSE37382", destdir = "data", GSEMatrix = TRUE)
```

Comment:
I think this all may be in the wrong spot? Based on our in person discussion, I suspect you did not move all the files to the new `validate-differential-expression` module.